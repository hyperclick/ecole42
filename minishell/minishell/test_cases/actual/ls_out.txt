1.txt
CMakeLists.txt
Makefile
author
debug_out.txt
libft
log.txt
minishell
src
test_cases
tests
ut
