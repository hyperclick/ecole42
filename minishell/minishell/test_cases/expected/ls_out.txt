ls:
CMakeLists.txt
Makefile
author
debug_out4.txt
dir  with  spaces
libft
log.txt
minishell
minishell.dSYM
not empty dir
not empty dir ls
push.sh
src
test_cases
tests
ut
ut.dSYM
not empty dir:
ls
dir1
file1
